backend/src/main/java/com/carffeine/carffeine/service/ScrapperService.java

```diff
@@ -27,11 +27,13 @@ public class ScrapperService {
     private static final String DATA_TYPE = "JSON";
     private final RestTemplate restTemplate;
     private final ChargeStationRepository chargeStationRepository;
-    @Value("${api.service_key}")
-    private String SERVICE_KEY;
+    private final RandomKeySelector randomKeySelector;
 
     @Transactional
     public void scrap() {
+
+        String SERVICE_KEY = randomKeySelector.getRandomKey();
```

Comment:
```suggestion
        String serviceKey = randomKeySelector.getRandomKey();
```
상수가 아니니깐 이게 맞지 않을까요?